src/tmqh-flow.c

```diff
@@ -98,6 +99,8 @@ Packet *TmqhInputFlow(ThreadVars *tv)
 
     StatsSyncCountersIfSignalled(tv);
 
+    TmThreadsSetFlag(tv, THV_RUNNING);
```

Comment:
another per packet atomic set for something that is essentially something that should only happen at startup